pages/common/gh-ssh-key.md

```diff
@@ -0,0 +1,20 @@
+# gh ssh-key
+
+> Manage GitHub SSH keys from the command line.
+> More information: <https://cli.github.com/manual/gh_ssh-key>.
+
+- Display help:
+
+`gh ssh-key`
+
+- List SSH keys:
+
+`gh ssh-key list`
+
+- Add an SSH key to the user:
+
+`gh ssh-key add {{path/to/key.pub}}`
+
+- Add an SSH key to the user with a specific title:
```

Comment:
```suggestion
- Add an SSH key to your account with a custom title:
```

Reply:
Why try to avoid her/him/your. Take a look to https://github.com/tldr-pages/tldr/blob/master/pages/common/gh-issue.md

Reply:
I known about him/her, but I didn't know about your.  Well we seem to use it in quite a few pages...126 of 2,221 (5.67%).  Still?

Reply:
I like 'your' because it gets the point across so clearly and without the slightest confusion.

Reply:
Hmm, I think using something like `the currently authenticated user` makes this quite readable without having `him/her/your`. I also think that `specific` works better than `custom`, but I don't mind either way. 👍🏻

Reply:
Personally I prefer custom in some circumstances just because it has fewer syllables.  But I think you're right.

Reply:
Generally speaking we try to avoid the 2nd person in tldr-pages. I like the idea of avoid him/her too - @owenvoke's suggestion here seems like the best solution.